apps/src/code-studio/pd/workshop_dashboard/components/session_form_part.jsx

```diff
@@ -125,37 +162,27 @@ var SessionFormPart = React.createClass({
         </Col>
         <Col sm={3}>
           <FormGroup validationState={style.startTime}>
-            <InputGroup>
-              <FormControl
-                type="time"
-                placeholder="hh:mm"
-                value={this.props.session.startTime || ''}
-                onChange={this.handleStartTimeChange}
-                style={this.props.readOnly && styles.readOnlyInput}
-                disabled={this.props.readOnly}
-              />
-              <InputGroup.Addon>
-                {!this.props.readOnly && <i className="fa fa-clock-o" />}
-              </InputGroup.Addon>
-            </InputGroup>
+            <TimeSelect
+              id="startTime-select"
+              onChange={value => this.handleChange('startTime', value)}
```

Comment:
Same comment about trying to avoid creating functions inside render methods when possible.